src/coreComponents/linearAlgebra/utilities/LinearSolverParameters.hpp

```diff
@@ -39,6 +39,20 @@ class LinearSolverParameters
   string preconditionerType = "iluk";  ///< Preconditioner type [none, iluk, ilut, amg, mgr, block]
   integer dofsPerNode = 1;             ///< Dofs per node (or support location) for non-scalar problems
   bool isSymmetric = false;            ///< Whether input matrix is symmetric (may affect choice of scheme)
+  integer stopIfError = 1;             ///< Whether to stop the simulation if the linear solver reports an error
+
+  /// Direct solver parameters
```

Comment:
We should clearly specify that the `struct` is of interest for the direct solver (`SuperLU_Dist`) used by _hypre_ and Petsc. They're irrelevant for `KLU` used by Trilinos.

Reply:
You're right, even if the tolerance could be used to check the Trilinos/KLU outcome (now it's not used).